File: THE_BROWN_BOX/00_LEARN1_CTDL & GIAI THUAT - BASIC/Java/8_Set.js
import java.util.HashSet;
import java.util.Set;
public class SetInJava {
    public static void main(String[] args) {
        //Tao 1 set kieu int
        Set<Integer> mySet = new HashSet<>();

        //2. add cac value vao set moi tao
        mySet.add(1);
        mySet.add(2);
        mySet.add(3);
        mySet.add(1);

        //In size of mySet
        System.out.print("mySet: " + mySet.size() + " ");
        //Duyet cac p.tu trong mySet
        for (Integer integer:mySet)
        {
            System.out.print(integer + " ");
        }

    }
}

File: THE_BROWN_BOX/00_LEARN1_CTDL & GIAI THUAT - BASIC/C++/8_Set.cpp
#include <iostream>
#include <set>
using namespace std;

int main()
{
    //Tao 1 set kieu int
    set<int> mySet;

    //2. add cac value vao set moi tao
    mySet.insert(1);
    mySet.insert(2);
    mySet.insert(3);
    mySet.insert(1);

    //In size of mySet
    cout << "mySet: " << mySet.size()<< endl;
    //Duyet cac p.tu trong mySet
    for (auto it = mySet.begin(); it != mySet.end(); it++)
    {
        cout << (*it) << " ";
    }
    return 0;
}
